app/models/answer.rb

```diff
@@ -0,0 +1,8 @@
+class Answer < ActiveRecord::Base
+  # Remember to create a migration!
+  belongs_to :question
+  belongs_to :answerer, class_name: "User"
```

Comment:
a better naming convention here could be `creator`. `answerer` is a little confusing